scenarios/k8s-scaling/run.sh

```diff
@@ -0,0 +1,18 @@
+# shellcheck disable=SC2046
+echo $(date +"%Y-%m-%d %T") start test
```

Comment:
```suggestion
echo "$(date +'%Y-%m-%d %T') start test"
```
Enclosing in " is not required? :o

Reply:
nope